microceph/ceph/config.go

```diff
@@ -3,58 +3,193 @@ package ceph
 import (
 	"context"
 	"database/sql"
+	"encoding/json"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/canonical/microceph/microceph/api/types"
 	"github.com/canonical/microceph/microceph/common"
 	"github.com/canonical/microceph/microceph/database"
 )
 
+type ConfigTable map[string]struct{
```

Comment:
Please add some doc on what the `ConfigTable` type is used for

Reply:
Ack